src/style.css

```diff
@@ -0,0 +1,87 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: Arial, sans-serif;
+  margin: 0;
+  padding: 0;
+}
+
+div {
+  display: block;
+}
+
+.blocks {
+  display: flex;
+  flex-wrap: wrap;
+  font-size: 100px;
+  text-align: center;
+  color: white;
+  line-height: 300px;
```

Comment:
Move these properties to .block class, it's not right to set style properties of children to wrapper class